### DISCUSSION TITLE
Questions about the lighthouse and the Python code

### DISCUSSION THREAD

**Discussion by sextta-feira** [bitcraze/discussions#1572] [Q&A] [answered]
After I calibrate the lighthouse and estimate the geometry in the Cfclient, as shown in the get started guide, and write this information to the Crazyflie, will my Python code already recognize this, or do I need to calibrate it again in my code? If it already recognizes it, is there any configuration I need to set? If not, how do I perform this calibration?

**Comment by gemenerik** [comment, marked as answer]
Once you calibrate the lighthouse and estimate the geometry in the Cfclient (as shown in the get started guide) and write this information to the Crazyflie, the geometry data is stored directly on the Crazyflie. Therefore, there's no need to redo the calibration/geometry estimation in your Python code — the Crazyflie will automatically recognize and apply the stored data.

In summary, there's nothing extra you need to configure in your Python code; it will automatically use the calibration you've already performed.
